overrides/android-override.json

```
@@ -1720,6 +1720,22 @@
         },
         "bookmarksSorting": {
             "state": "disabled"
+        },
+        "settingsScreen": {
```

Can we name it settings-page instead as we already have new-tab-page and other platforms appear to be calling it settings-page too?

```suggestion
        "settingsPage": {
```